solutions/src/test/java/ru/sberbank/school/task10/MyScalableConcurrentThreadPoolTest.java

```diff
@@ -0,0 +1,208 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.Queue;
+import java.util.concurrent.Callable;
+import java.util.concurrent.FutureTask;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MyScalableConcurrentThreadPoolTest {
+    private MyScalableConcurrentThreadPool pool;
+
+    private void fillScalablePool() {
+        pool = new MyScalableConcurrentThreadPool(4, 10);
+        }
+
+    private void fillFixedPool() {
+        pool = new MyScalableConcurrentThreadPool(2);
+        }
+
+    private void fillTasks() {
+        Runnable runnable = () -> {
+        try {
+        Thread.sleep(50);
+        } catch (InterruptedException e) {
+        e.printStackTrace();
+        }
+        System.out.println(Thread.currentThread() + " running");
+        };
+
+        Callable<Integer> callable = () -> {
+        try {
+        Thread.sleep(50);
+        } catch (InterruptedException e) {
+        e.printStackTrace();
+        }
+        System.out.println(Thread.currentThread() + " running");
+        return null;
+        };
+
+        for (int i = 0; i < 10; i++) {
+        pool.execute(runnable);
+        }
+
+        for (int i = 0; i < 10; i++) {
+        pool.execute(callable);
```

Comment:
&#39;for&#39; child has incorrect indentation level 8, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a878655083>*